src/DependencyInjection/Configuration.php

```diff
@@ -52,6 +52,22 @@ public function getConfigTreeBuilder(): TreeBuilder
             ->booleanNode('enable_twig')
                 ->info('Whether or not to enable the twig `request_id()` function. Only works if TwigBundle is present.')
                 ->defaultTrue()
+            ->end()
+            ->arrayNode('http_client')
+                ->children()
+                    ->booleanNode('enabled')
+                        ->info('Whether or not to enable the request id aware http client')
+                        ->defaultTrue()
```

Comment:
Set the default to false. Forwarding this information "accidentally" to unwanted http clients should not happen.